tests/integration/generic_test.go

```diff
@@ -116,13 +117,11 @@ var _ = Describe("odo generic", func() {
 			reOdoVersion := regexp.MustCompile(`^odo\s*v[0-9]+.[0-9]+.[0-9]+(?:-\w+)?\s*\(\w+\)`)
 			odoVersionStringMatch := reOdoVersion.MatchString(odoVersion)
 			Expect(odoVersionStringMatch).Should(BeTrue())
-			if !helper.IsKubernetesCluster() {
-				rekubernetesVersion := regexp.MustCompile(`Kubernetes:\s*v[0-9]+.[0-9]+.[0-9]+((-\w+\.[0-9]+)?\+\w+)?`)
-				kubernetesVersionStringMatch := rekubernetesVersion.MatchString(odoVersion)
-				Expect(kubernetesVersionStringMatch).Should(BeTrue())
-				serverURL := oc.GetCurrentServerURL()
-				Expect(odoVersion).Should(ContainSubstring("Server: " + serverURL))
-			}
+			rekubernetesVersion := regexp.MustCompile(`Kubernetes:\s*v[0-9]+.[0-9]+.[0-9]+((-\w+\.[0-9]+)?\+\w+)?`)
+			kubernetesVersionStringMatch := rekubernetesVersion.MatchString(odoVersion)
+			Expect(kubernetesVersionStringMatch).Should(BeTrue())
```

Comment:
Could you use the [`MatchRegexp`](https://onsi.github.io/gomega/#matchregexpregexp-string-args-interface) matcher instead? In case the test no longer passes, I think the output would be easier to analyze.

Also, for better readability, maybe you could also combine both requirements in a single composite matcher, like so:
```go
Expect(odoVersion).Should(SatisfyAll(
   MatchRegexp(regexp1),
   MatchRegexp(regexp2)))
```